iOS/Controllers/Dashboard/DashboardNoPermissionViewController.swift

```diff
@@ -17,7 +17,6 @@ final class DashboardNoPermissionViewController: UIViewController, ViewHeightDat
     }
 
     var height: CGFloat {
-        return 200.0
+        return self.label.yMax
```

Comment:
Why not `self.label.height`?

Reply:
yMax does `origin.y + size.height` taking into the origin margin. Using `self.eventOverview.layoutMargins`